### PR TITLE
upgrade to sbt 1.0

### PR DESCRIPTION
**Pull request by godenji**

### CODE REVIEW

**Comment by jrudolph** [comment]
Thanks, @godenji. Thanks for helping out. Seemed simple enough. I'll try to make a release when I get to it.

**Comment by godenji** [comment]
Sure, no rush, just needed to take the project for a spin with sbt 1.0, so made the necessary changes.

File: publish.sbt [hunk 1]
```diff
@@ -1,7 +1,7 @@
-publishTo <<= version { v: String =>
```
Comment (jozic) [inline publish.sbt]: this also should work
```scala
publishTo := Some(
  if (isSnapshot.value)
    Opts.resolver.sonatypeSnapshots
  else
    Opts.resolver.sonatypeStaging
```

**Comment by atais** [comment]
just found this project and I am really looking forward to sbt 1.x release

**Comment by jrudolph** [comment]
Finally merged it, thanks a lot @godenji.

**Comment by godenji** [comment]
Completely forgot about this PR, was puzzled as to what I was being thanked for :)

**Pull request merged**